### DISCUSSION TITLE
[BUG] 在线预览大文件log日志导致浏览器崩溃

### DISCUSSION THREAD

**Discussion by XM2510136957** [1Panel-dev/1Panel#5653] [Q&A]
### 联系方式

2510136957@qq.com

### 1Panel 版本

v1.10.11-lts

### 问题描述

在线预览大文件log日志导致浏览器崩溃

### 重现步骤

在线预览大文件log日志导致浏览器崩溃

### 附加信息

![image](https://github.com/1Panel-dev/1Panel/assets/18229485/eea1a375-4a64-499a-9f20-fac784f69c83)

**Comment by wanghe-fit2cloud** [comment]
麻烦提供一下日志文件。

**Reply by XM2510136957** [reply]
![image](https://github.com/1Panel-dev/1Panel/assets/18229485/86a104e1-092d-4c27-9b4c-2d080d3fdf89)
守护进程里面查询日志的时候，网页卡顿，然后内存溢出了！

**Reply by XM2510136957** [reply]
![image](https://github.com/1Panel-dev/1Panel/assets/18229485/e9c97182-dff7-456c-b9fd-45353b138f37)
![image](https://github.com/1Panel-dev/1Panel/assets/18229485/fa06d062-ca1f-4d8d-b1aa-50c443a5085b)
![image](https://github.com/1Panel-dev/1Panel/assets/18229485/22f54dbf-90ff-45fa-b7f5-6800279746a9)
百分百复现

**Reply by wanghe-fit2cloud** [reply]
1、将该日志文件在 `主机` - `文件管理` 页面打开，是否存在浏览器崩溃的情况。
2、麻烦提供一下该日志文件，我们在本地环境复现一下。

> 刚才在进程守护页面查看日志并未能复现上述问题。
